models/rne/types.ts

```diff
@@ -23,33 +23,49 @@ export interface IEtatCivil {
   prenom: string;
   prenoms: string;
   role: string;
+  roles?: IRole[];
   lieuNaissance: string;
   dateNaissancePartial?: string;
   dateNaissance?: string;
   nationalite?: string;
+  isInInpi?: boolean;
+  isInIg?: boolean;
 }
 
 export interface IPersonneMorale {
   siren: string;
   denomination: string;
   natureJuridique: string | null;
   role: string;
+  roles?: IRole[];
+  isInInpi?: boolean;
+  isInIg?: boolean;
 }
 
-export interface IObservations {
-  data: {
-    numObservation: string;
-    dateAjout: string;
-    description: string;
-  }[];
+export type IObservations = {
+  numObservation: string;
+  dateAjout: string;
+  description: string;
+}[];
+
+export interface IObservationsWithMetadata {
```

Comment:
A bit lost on this `WithMetadata` as I would expect  `IObservations` to always have metadata as they always come from API Proxy

Reply:
We often used IDirigeants["data"] and rarely (if ever) IDirigeants, so it was more convenient this way.
Also, the data inside IDirigeant doesn't always come with metadata.

I made the same change for both IObservations and IDirigeants because they are close in the code and it is coherent.

We could indeed keep IObservations rather than IObservationsWithMetadata.